Bad sourcemap error

Recently when I deleted my node modules and reinstalled them, I got the following error from rollup even though my npm package.json and rollup.config.js remain the unchanged. There doesn't seem to be any reference of error anywhere.

`Bad sourcemap
Error: Bad sourcemap
    at Object.Link.traceSegment (xxx\xxx\xxx\node_modules\rollup\www\ROLLUP\rollup\src\utils                                                                                                                                         \collapseSourcemaps.js:75:26)
    at Object.Link.traceSegment (xxx\xxx\xxx\node_modules\rollup\www\ROLLUP\rollup\src\utils                                                                                                                                         \collapseSourcemaps.js:77:19)
    at Object.Link.traceSegment (xxx\xxx\xxx\node_modules\rollup\www\ROLLUP\rollup\src\utils                                                                                                                                         \collapseSourcemaps.js:77:19)
    at xxx\xxx\xxx\node_modules\rollup\www\ROLLUP\rollup\src\utils\collapseSourcemaps.js:31:                                                                                                                                         27
    at Array.forEach (native)
    at xxx\xxx\xxx\node_modules\rollup\www\ROLLUP\rollup\src\utils\collapseSourcemaps.js:29:                                                                                                                                         9
    at Array.map (native)
    at Object.Link.traceMappings (xxx\xxx\xxx\node_modules\rollup\www\ROLLUP\rollup\src\util                                                                                                                                         s\collapseSourcemaps.js:26:34)
    at collapseSourcemaps (xxx\xxx\xxx\node_modules\rollup\www\ROLLUP\rollup\src\utils\colla                                                                                                                                         pseSourcemaps.js:102:37)
    at Bundle.render (xxx\xxx\xxx\node_modules\rollup\www\ROLLUP\rollup\src\Bundle.js:280:11                                                                                                                                         )
Type rollup --help for help, or visit https://github.com/rollup/rollup/wiki

npm ERR! Windows_NT 10.0.10240
npm ERR! argv "E:\\Program Files\\nodejs\\node.exe" "E:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "start"
npm ERR! node v4.4.3
npm ERR! npm  v2.11.3
npm ERR! code ELIFECYCLE
npm ERR! webserver@1.0.0 start: `rollup -c rollup.config.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the webserver@1.0.0 start script 'rollup -c rollup.config.js'.
npm ERR! This is most likely a problem with the webserver package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     rollup -c rollup.config.js
npm ERR! You can get their info via:
npm ERR!     npm owner ls webserver
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     xxx\xxx\xxx\npm-debug.log
`

**My rollup.config.js file is as follow:**
`import babel from 'rollup-plugin-babel';
import commonjs from 'rollup-plugin-commonjs';
import nodeResolve from 'rollup-plugin-node-resolve';
import replace from 'rollup-plugin-replace';

export default {
  entry: 'static/js/main.js',
  sourceMap: true,
  plugins: [
    babel(),
    nodeResolve(),
    commonjs(),
    replace({'process.env.NODE_ENV': JSON.stringify(process.env.NODE_ENV)})
    ],
  dest: 'static/bundle.js'
};`

Does anyone have an idea of what might have happened here?

--- a/src/utils/collapseSourcemaps.js
+++ b/src/utils/collapseSourcemaps.js
@@ -71,8 +71,7 @@ Link.prototype = { // TODO bring into line with others post-https://github.com/r
 
 			if ( segment[0] === column ) {
 				const source = this.sources[ segment[1] ];
-
-				if ( !source ) throw new Error( 'Bad sourcemap' );
+				if ( !source ) return null;
 
 				return source.traceSegment( segment[2], segment[3], this.names[ segment[4] ] || name );
 			}

--- a/test/sourcemaps/single-length-segments/_config.js
+++ b/test/sourcemaps/single-length-segments/_config.js
@@ -1,0 +1,37 @@
+var fs = require( 'fs' );
+var path = require( 'path' );
+var assert = require( 'assert' );
+var getLocation = require( '../../utils/getLocation' );
+var SourceMapConsumer = require( 'source-map' ).SourceMapConsumer;
+
+var original = fs.readFileSync( path.resolve( __dirname, 'main.js' ), 'utf-8' );
+
+module.exports = {
+	description: 'handles single-length sourcemap segments',
+	options: {
+		plugins: [
+			{
+				transform: function () {
+					return {
+						code: fs.readFileSync( path.resolve( __dirname, 'output.js' ), 'utf-8' ),
+						map: fs.readFileSync( path.resolve( __dirname, 'output.js.map' ), 'utf-8' )
+					};
+				}
+			}
+		],
+		moduleName: 'x'
+	},
+	test: function ( code, map ) {
+		var smc = new SourceMapConsumer( map );
+
+		[ 'Foo', 'log' ].forEach( function ( token ) {
+			var generatedLoc = getLocation( code, code.indexOf( token ) );
+			var originalLoc = smc.originalPositionFor( generatedLoc );
+			var expectedLoc = getLocation( original, original.indexOf( token ) );
+
+			assert.ok( /main/.test( originalLoc.source ) );
+			assert.equal( originalLoc.line, expectedLoc.line );
+			assert.equal( originalLoc.column, expectedLoc.column );
+		});
+	}
+};

--- a/test/sourcemaps/single-length-segments/main.js
+++ b/test/sourcemaps/single-length-segments/main.js
@@ -1,0 +1,7 @@
+class Foo {
+	bar () {
+		console.log( 42 );
+	}
+}
+
+export { Foo };

--- a/test/sourcemaps/single-length-segments/output.js
+++ b/test/sourcemaps/single-length-segments/output.js
@@ -1,0 +1,15 @@
+var Foo = function () {
+	function Foo() {
+		babelHelpers.classCallCheck(this, Foo);
+	}
+
+	babelHelpers.createClass(Foo, [{
+		key: "bar",
+		value: function bar() {
+			console.log(42);
+		}
+	}]);
+	return Foo;
+}();
+
+export { Foo };

